tests/io/datasets/test_seq.py

```diff
@@ -1,5 +1,153 @@
+import os
+
+import numpy as np
+import pytest
+
 from libertem.executor.inline import InlineJobExecutor
 from libertem.io.dataset.seq import SEQDataSet
+from libertem.common import Shape
+from libertem.io.dataset.base import TilingScheme
+
+SEQ_TEST_PATH = os.path.join(os.path.dirname(__file__), '..', '..', '..', 'data', 'default.seq')
```

Comment:
Really good to have these tests, also the ones for the client-side code!

It would be good to add a test that hits the "offset should be in (...)" exception (both the min and max case would be great). To find missing coverage cases like this one, it helps to look at the coverage report generated in the "htmlcov" directory (generated when running the tests via tox, or manually via `pytest --cov=libertem --cov-report=html ...`